terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -124,14 +124,9 @@ locals {
       "output_file_prefix": null,
       "output_directory": null,
       "fastq_list_rows": null,
-      "sites_somalier": {
```

Comment:
Right! So this removed somalier step at workflow side now. Got it!

Reply:
Which reminds me I need to do the same for transcriptome workflow as well. 
@alexiswl - should we submit the changes for that in this same PR or a new one? 
I am happy with both options.

Reply:
Might as well do it now, do you need to update the workflow as well. That shouldn't take too long.

Reply:
Yes, I am on it. Will ping you on PR in `cwl-ica` in a tick.